File: PraticeAnyLanguage/test3.c
#include <stdio.h>

double factorial(int n);

int main()
{
    int n;
    printf("Enter a number to get it's factorial:\n");
    scanf("%d", &n);
    printf("The factorial of %d is %f", n, factorial(n));
    return 0;
}

double factorial(int n)
{
    if (n == 0 || n == 1)
        return 1;
    else
        return n * factorial(n - 1);
}

File: CODEJAVA/allTestCodes/test3.java
// BufferReader input.
package allTestCodes;

import java.io.*;

public class test3 
{
    public static void main(String[] args) throws IOException
    {
        double list[] = new double[5];
        
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);
        
        System.out.println("Enter 5 number:");
        for(int i = 0; i < 5; i++){
            String input = br.readLine();
            list[i] = Double.parseDouble(input);
        }
        System.out.println("Your number are:");
        for(int i = 0; i < 5; i++)
        {
            System.out.println(list[i]);
        }
    }
}